lib/oci.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
+
+export class OciUploader {
+
+    constructor(url, file, maxFileSizeChunking, emit) {
+
+        this.url = url;
+        this.file = file;
+        this.emit = emit;
+        this.reader = new FileReader();
+        this.blockIds = [];
+        this.failOnBlock = false;
+        this.currentFilePointer = 0;
+        this.totalBytesRemaining = this.file.size;
+        this.percentComplete = 0;
+        this.bytesUploaded = 0;
+        this.maxBlockSize = 5242880;  
+        this.timeStarted;
+        this.retryChunkUploadLimit = 10;
+        this.delayBetweenRetryCalls = 2000;
+        if(maxFileSizeChunking < 6 || !maxFileSizeChunking) {
+            maxFileSizeChunking = 6
+        }
+        if (parseInt(this.file.size / 1e+6) <= maxFileSizeChunking) {
```

Comment:
Since here there is no file chunk uploads this condition can be removed

Reply:
I have added if condition to allow maximum upload file size.